### ISSUE TITLE
Einthusan Not working in kodi

### ISSUE THREAD

**Issue by glardz98** [eintamil/repository.eintamil#44]
Hi,

I use einthusan addon and its Not working.

**Comment by hamid1135** [comment]
the same thing happens to me

**Comment by vjayapau** [comment]
Please remove your Einthusan logon credentials from the add-on and use default(none).
It worked for me.
When I entered my credential, the issue came back again.
So I am using it with no credentials.
Let know if it worked.

**Comment by glardz98** [comment]
I prefer to use with credential, do you have other solution ?

**Comment by vjayapau** [comment]
No and I am not sure whether the eintamil team has either.
You are not loosing anything by not using the credential. So what is the big deal?

**Comment by ottsubs** [comment]
Attached is the screen capture of the failure log message. When disabled the credentials, i am able to play videos, but not when using premium login credentials.
![Einthusan_Error](https://github.com/user-attachments/assets/78b76033-2608-4d2a-9875-611c4658ae67)

**Comment by alakssevugan** [comment]
In the error message, I found that the **EJlinks** field was not found in the Json content returned from the following postdata action

    postdata = {
        "xEvent": "UIVideoPlayer.PingOutcome",
        "xJson": jdata,
        "arcVersion": "3",
        "appVersion": "59",
        "gorilla.csrf.Token": csrf1,
    }

    rdata = s.post(videourlajax, headers=headers, data=postdata, cookies=s.cookies).text
    addLog(rdata)
    **ejl = json.loads(rdata)["Data"]["EJLinks"]**

**Comment by eintamil** [comment]
Please try the below steps and comment if you are still facing the issue:
```
- Kindly uninstall the older version of the Einthusan addon
- Update to EinTamil Kodi Repository Version 1.1.0
- Install Einthusan addon Version 3.1.0
```

**Comment by alakssevugan** [comment]
I’m using version 3.1.0 version and am getting the same error - line 480, in get_video method.

**Comment by eintamil** [comment]
Is your login a premium user account or regular account?

Please uninstall the addon (also remove addon data).
Restart the device and install the addon.
If the issue still exists, please post the log file.

**Comment by glardz98** [comment]
I use premium account, and i reinstall addon but Not working, same error.

**Comment by rajdil** [comment]
I am getting this even after updating to version 3.10. 
![image](https://github.com/user-attachments/assets/b6a0c1a3-3dce-445e-bdbd-02858ed5e953)

**Comment by rajdil** [comment]
without login it plays , but some time it errors as server busy hence it is better to fix the issue

**Comment by eintamil** [comment]
@rajdil please share few more logs before this error

**Comment by eintamil** [comment]
@rajdil Issue resolved in Version 3.1.1
